modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -349,41 +332,37 @@ class Configuration(object):
             {{/servers}}
         ]
 
-    def get_host_from_settings(self, index, variables={}):
-        """Gets host URL based on the index and variables
-        :param index: array index of the host settings
-        :param variables: hash of variable and the corresponding value
+    def get_host_from_settings(self, index=0, variables=None):
```

Comment:
Or we can keep it without a default value.
```suggestion
    def get_host_from_settings(self, index, variables=None):
```

Reply:
While I like this default, adding it will break legacy code because it was a positional argument before and we are changing it to a named argument.
Can we remove the default and keep it as positional?
Sorry for the new different guidance here. More thought on it led me to the above conclusion.